### PR TITLE
Add Cypress to Jenkins

### PR DESCRIPTION
**Pull request by theodur**
## Description
This PR adds running Cypress tests to the `e2e` step in Jenkins.

Keeping this as a draft until we get some Cypress tests merged in because `cypress run` fails if no tests are found.

## Testing done
Tested on another [branch](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fvets-website/detail/convert-hca-tests-to-cypress/148/pipeline/153).

## Acceptance criteria
- [ ] Cypress runs properly on the `e2e` step in Jenkins.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "build:content:test": "node script/test-cms-export.js",
     "build:webpack": "NODE_OPTIONS=--max-old-space-size=4096 webpack --config config/webpack.config.js --progress",
     "cy:test:circle": "./script/run-cypress-tests-circle.sh",
+    "cy:test:docker": "cypress run --config baseUrl=http://vets-website:3001",
```
Comment (U-DON) [inline package.json:14]: Don't we also want to pass the globbed pattern to match the tests?

Comment (U-DON) [reply package.json:14]:
```suggestion
    "cy:test:docker": "cypress run --config baseUrl=http://vets-website:3001 --spec 'src/**/tests/**/*.cypress.spec.js'",
```

Comment (theodur) [reply package.json:14]: Wouldn't the config in `cypress.json` be used for determining the `testFiles`?

Comment (U-DON) [reply package.json:14]: Oh, that's right. I forgot that was configured in there.

**Comment by U-DON** [comment]
Why didn't the CircleCI setup fail from finding no tests? What was the difference?

**Comment by theodur** [comment]
https://github.com/department-of-veterans-affairs/vets-website/blob/a0a03996fe8d0bb695b7a96fe30a6a9911afb7bf/script/run-cypress-tests-circle.sh#L3

For the CircleCI setup I made a shell script for running the command, and added an `if` statement that checks if there are any Cypress tests before running. I could make separate script for Jenkins too, I just wasn't sure if it was needed if we are holding off on integrating Jenkins until Cypress has made it past phase 1.

**Review by U-DON — APPROVED** [review]

**Pull request merged**